src/Tizen.NUI/src/internal/Common/Registry.cs

```diff
@@ -82,13 +82,13 @@ internal static bool Register(BaseHandle baseHandle)
                     var target = weakReference.Target;
 
                     BaseHandle ret = target as BaseHandle;
-                    if ((ret == null) || ret.Disposed || ret.IsDisposeQueued)
+                    // Note : Do not use ret == null because BaseHandle override operator ==.
```

Comment:
It's really important for the future!
Thanks to add it and I hope everyone know this note !